### PR TITLE
feat(frontend): Button with popup

### PR DESCRIPTION
**Pull request by adam-kov**
desktop

---
destop2

Clicking or targeting the toggle button with the Tab key will open the popup. Tab targeting inside the opened window is disabled, keyboard navigation is only available with the arrow keys, so the user doesn't have to tab through the list of actions if they want to target the next element.

### CODE REVIEW

**Comment by rubenfiszel** [comment]
Looks great. Another great use of it would be on RunForm to schedule for later from the run button

**Comment by fatonramadani** [comment]
Great work! Having the popup floating feels a little bit odd.
What about adding a border to the popup ? 

You can have a nice example here:
https://web.dev/building-a-split-button-component/

Material UI also uses a menu:
https://mui.com/material-ui/react-button-group/#split-button

The keyboard hint could be displayed in a footer inside the menu.

File: frontend/src/lib/components/common/popup/Popup.svelte [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+	export const ref = popperRef
+	export const toggle = () => (isOpen ? close() : open())
+	export const open = () => (isOpen = true)
+	export const close = () => (isOpen = isFocused = false)
+
+	let element: HTMLElement
+	let focusableElements: HTMLElement[]
+
+	const getFocusableElements = () => {
```
Comment (fatonramadani) [inline frontend/src/lib/components/common/popup/Popup.svelte]: To stay consistent with the codebase, I prefer we use the `function` syntax.

**Comment by adam-kov** [comment]
Updated the popup window. What do you think about the shadow, should it be more subtle?

desktop

**Comment by fatonramadani** [comment]
> 

This is great! What do you think on having the items align to the left ?

**Comment by adam-kov** [comment]
> This is great! What do you think on having the items align to the left ?

Yea, it would look better if there are icons on the left.

**Review by fatonramadani — APPROVED** [review]

**Review by fatonramadani — APPROVED** [review]

**Pull request merged**